crates/client/src/client.rs

```diff
@@ -358,26 +361,33 @@ impl FuelClient {
     }
 
     /// Default dry run, matching the exact configuration as the node
-    pub async fn dry_run(&self, tx: &Transaction) -> io::Result<Vec<Receipt>> {
-        self.dry_run_opt(tx, None).await
+    pub async fn dry_run(
+        &self,
+        txs: &[&Transaction],
```

Comment:
Do you expect to already have a `Vec<&Transaction>`? I feel like this could just be `&[Transaction]`.

Reply:
hmm I think I did try to use `&[Transaction]` at first, but I was running into some errors.  I'll take another look at it.

Reply:
Happy to look at it with you.

Reply:
lol nvm, `&[Transaction]` totally works.  Thanks for the feedback :)